docs/index.md

```diff
@@ -1,10 +0,0 @@
----
-layout: simple
-title: Home
-section: Home
----
-
-Style Guide
```

Comment:
there needs to be an index page so when you land on the site (https://daptiv.github.io/style-guide/) it can show a page instead of a directory listing.